solitude/settings/base.py

```diff
@@ -19,14 +19,6 @@
 #
 ALLOWED_HOSTS = []
 
-if os.environ.get('MEMCACHE_URL'):
-    CACHES = {
-        'default': {
-            'BACKEND': 'django.core.cache.backends.memcached.MemcachedCache',
-            'LOCATION': os.getenv('MEMCACHE_URL'),
-        }
-    }
-
```

Comment:
I think we should leave this in just in case someone needs to reproduce a memcache related issue locally. Also, a quick look at prod settings tells me that it would rely on this env detection if it were to run with memcache.